dungeon/src/contrib/level/generator/graphBased/levelGraph/LevelNode.java

```diff
@@ -69,6 +69,8 @@ public LevelNode(final LevelGraph originGraph) {
    * @return true if the connection was successful, false if not.
    */
   public boolean connect(final LevelNode other) {
+    // This method should have public access modifier (like the other methods in this class) because
+    // it is used to connect own LevelNodes or graphs.
```

Comment:
@tgrothe Dieser Kommentar hat hier eigentlich gar nichts zu suchen - ich hatte Dich gebeten, den ersten Kommentar des Pull-Requests zu ergänzen. 

Der erste Kommentar des PR geht als Commit-Message mit in das Git Log - und dort möchte ich erkennen, _warum_ eine bestimmte Änderung erfolgt ist.

Reply:
Und wenn ich grad nochmal drauf schaue: Die Sichtbarkeit wurde doch gar nicht geändert in diesem PR, oder? Also musst Du doch hier gar nichts extra erklären?